src/libraries/NormalizedAmount.sol

```diff
@@ -3,6 +3,40 @@ pragma solidity >=0.8.0 <0.9.0;
 
 type NormalizedAmount is uint64;
 
+using {gt as >, lt as <, sub as -, add as +, div, mul, min, unwrap} for NormalizedAmount global;
+
+function gt(NormalizedAmount a, NormalizedAmount b) pure returns (bool) {
+    return NormalizedAmount.unwrap(a) > NormalizedAmount.unwrap(b);
+}
+
+function lt(NormalizedAmount a, NormalizedAmount b) pure returns (bool) {
+    return NormalizedAmount.unwrap(a) < NormalizedAmount.unwrap(b);
+}
+
+function sub(NormalizedAmount a, NormalizedAmount b) pure returns (NormalizedAmount) {
+    return NormalizedAmount.wrap(NormalizedAmount.unwrap(a) - NormalizedAmount.unwrap(b));
+}
+
+function add(NormalizedAmount a, NormalizedAmount b) pure returns (NormalizedAmount) {
+    return NormalizedAmount.wrap(NormalizedAmount.unwrap(a) + NormalizedAmount.unwrap(b));
+}
+
+function div(NormalizedAmount a, uint64 b) pure returns (NormalizedAmount) {
+    return NormalizedAmount.wrap(NormalizedAmount.unwrap(a) / b);
+}
+
+function mul(NormalizedAmount a, uint64 b) pure returns (NormalizedAmount) {
+    return NormalizedAmount.wrap(NormalizedAmount.unwrap(a) * b);
+}
+
+function min(NormalizedAmount a, NormalizedAmount b) pure returns (NormalizedAmount) {
+    return a > b ? b : a;
+}
+
+function unwrap(NormalizedAmount a) pure returns (uint64) {
```

Comment:
Do we need this `unwrap` method? I thought it was implicit?

Reply:
it just allows `foo.unwrap()` instead of `NormalizedAmount.unwrap(foo)` (but it's never done implicitly, that's the point!)

Reply:
Sorry yeah implicit was the wrong word! That's nice though, it looks a bit cleaner this way